package.json

```diff
@@ -15,6 +15,7 @@
     "start": "node src/app.js",
     "lint": "eslint .",
     "test": "istanbul test -- _mocha",
+    "test-oracle": "LEDGER_DB_ENV=oracledev DYLD_LIBRARY_PATH=/opt/oracle/instantclient node node_modules/.bin/istanbul test -- _mocha",
```

Comment:
Ok, I think I know what's going on here. I made a file "test.sh" containing:
```
#!/usr/bin/env node

console.log(process.env.DYLD_LIBRARY_PATH)
```

```
bash-3.2$ chmod +x ./test.sh
bash-3.2$ DYLD_LIBRARY_PATH=HI ./test.sh
undefined
bash-3.2$ DYLD_LIBRARY_PATH=HI node ./test.sh
HI
```

I think for some security reason certain applications on OSX ignore environment variables starting with "DYLD_". Apparently `bash` is one of them, but `node` is not.